app/views/layouts/sumofus.slim

```diff
@@ -2,18 +2,22 @@ doctype html
 html
   head
     title  = @page.title
-    = render partial: 'shared/optimizely_snippet'
```

Comment:
I have experienced the problem of Optimizely flicker based on snippet location before, so agree with Tuuli :)